docs/devel/networking.md

```diff
@@ -72,6 +72,74 @@ As of now, the existent binding mechanisms are:
 - [slirp](#slirp-binding-mechanism)
 
 ### Bridge binding mechanism
+Using the bridge `BindMechanism` requires a VMI configuration featuring a
+network whose interface type is `bridge` - the yaml file below can be used
+as reference, but please refer to the
+[user guide](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=bridge)
+for more information.
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: default
+          bridge: {}
+  networks:
+  - name: default
+    pod: {} # Stock pod network
+```
+
+Let's refer to the image below to get a better understanding of how this
+`BindMechanism` works.
+
+Bridge binding mechanism diagram
+
+![alt text](https://github.com/kubevirt/kubevirt.github.io/blob/source/assets/images/diagram.png)
+
+As can be seen in the diagram above, there are three actors at play: CNI,
+libvirt, and DHCP. For completeness sake, let's add one more actor that is
+implicit in the picture: KubeVirt.
+
+Once the bridge `BindMechanism` is triggered,
+[CNI](https://github.com/containernetworking/plugins/tree/master/plugins/main/bridge)
+will create / configure a linux bridge in the k8s node. It will also create a
+veth pair, attaching one end of it to the bridge, and move the other end into
+the pod.
+
+KubeVirt, in the
+[preparePodNetworkInterfaces method](https://github.com/kubevirt/kubevirt/blob/5995d2164faefdcc7a59a857e12f8ed4cdb7e094/pkg/virt-launcher/virtwrap/network/podinterface.go#L323),
+updates the in-pod veth mac-address, creates an in-pod bridge, and connects
```

Comment:
> updates the in-pod veth mac-address

Why is this needed?

Reply:
Because we'll copy that MAC & "move" it to the VM interface. 

Is it clearer now ?